bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/StreamCombiningIterator.java

```diff
@@ -57,6 +66,11 @@ public class StreamCombiningIterator implements Iterator<ReadRowsResponse> {
   Object last;
   volatile boolean cancelled = false;
   private final Collection<Observer> observers;
+  private static final long IDENTITY_TOKEN_TTL_IN_MILLIS = 50 * 60 * 1000;
+  private static final LoadingCache<String, Optional<String>> READ_SESSION_TO_IDENTITY_TOKEN_CACHE =
+      CacheBuilder.newBuilder()
+          .expireAfterWrite(IDENTITY_TOKEN_TTL_IN_MILLIS, TimeUnit.MILLISECONDS)
```

Comment:
Notice that `.expireAfterWrite(IDENTITY_TOKEN_TTL_IN_MINUTES, TimeUnit.MINUTES)` is more readable

Reply:
Fixed